lib/r/extract_evaluate.dart

```diff
@@ -171,6 +151,26 @@ String _basicTemplate(
         '$pcf\n\n';
   }
 
+  // Append ADABOOST model results if available and boosting is enabled.
```

Comment:
Having all this model specific code here looks like a maintenance nightmare for me in the future.